occurrences/schema.py

```diff
@@ -475,7 +516,7 @@ class StudyGroupInput(InputObjectType):
     group_name = graphene.String()
     extra_needs = graphene.String()
     amount_of_adult = graphene.Int()
-    study_level = StudyLevelEnum()
+    study_levels = graphene.List(StudyLevelInput)
```

Comment:
You might not want to require all fields of StudyLevelInput in this mutation. I think we can only need study_level id here.
So it could be something like:
`study_levels = graphene.List(String)`
And also the `def _get_study_levels()` will become a bit simpler